src/assets/YAML/default/TestAndVerification/Consolidation.yaml

```diff
@@ -274,17 +283,23 @@ Test and Verification:
       comments: ""
     Treatment of defects with severity high or higher:
       uuid: 44f2c8a9-4aaa-4c72-942d-63f78b89f385
-      risk: Vulnerabilities with severity high or higher are not visible.
-      measure:
-        Vulnerabilities with severity high or higher are added to the quality
-        gate.
+      description: |
+        All security problems that are rated as "high" or "critical" must be fixed before the software can be released or used in production. This means that if a serious vulnerability is found, it cannot be ignored or postponed.
```

Comment:
The old text is not clear but leaves interpretation what "quality gate" is.

For known vulnerabilities, which one is better? Scanning for known vuln in prod or in the pipeline?
From my point of view scanning for known vuln. in prod is much more important than somewhere in a pipeline.

The problem arises because we do not have an extra SCA sub-dimension. It might be time to add it (not in this pr)?